Add dependencies so we can safely modify the cssmin.py, jsmin.py, or xxd.pl scripts

darinadler opened the pull request:
#### 9c70666dba64046e4aa38d96732b3ff87caa8314
<pre>
Add dependencies so we can safely modify the cssmin.py, jsmin.py, or xxd.pl scripts
<a href="https://bugs.webkit.org/show_bug.cgi?id=299664">https://bugs.webkit.org/show_bug.cgi?id=299664</a>
<a href="https://rdar.apple.com/161480706">rdar://161480706</a>

Reviewed by Yusuke Suzuki.

* Source/WebCore/DerivedSources-input.xcfilelist: Added cssmin.py and jsmin.py.
* Source/WebCore/DerivedSources.make: Added dependencies on cssmin.py, jsmin.py, and xxd.pl.

Canonical link: <a href="https://commits.webkit.org/300643@main">https://commits.webkit.org/300643@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/7ae8afa67a220d40ee3d72615a8038b931dcb489

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/123333 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/43048 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/33744 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/130046 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/75452 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/e0cc7fd8-c773-4e7b-8187-4c6d635f26dc) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/125210 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/43771 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/51642 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/93749 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/62194 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/68494cac-72a2-40b9-b08e-100e61e22c0e) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/126286 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/132/builds/34876 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/110355 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/74379 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/1ee9f22a-92c3-4c7f-b60d-2c1d74719884) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/33842 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/28514 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/73562 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/104590 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/28740 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/132762 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/50283 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/38264 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/102240 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/50659 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/106579 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/102093 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25956 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/47452 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/25677 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/47069 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/50138 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/55899 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/49609 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/52959 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/51287 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

Constellation reviewed (approved):
Looks good! It seems that CMake side is already setting up dependencies, so this looks fine.

darinadler committed:
Add dependencies so we can safely modify the cssmin.py, jsmin.py, or xxd.pl scripts
https://bugs.webkit.org/show_bug.cgi?id=299664
rdar://161480706

Reviewed by Yusuke Suzuki.

* Source/WebCore/DerivedSources-input.xcfilelist: Added cssmin.py and jsmin.py.
* Source/WebCore/DerivedSources.make: Added dependencies on cssmin.py, jsmin.py, and xxd.pl.

Canonical link: https://commits.webkit.org/300643@main
